examples/jeventdemo/source/src/org/kaaproject/kaa/demo/event/EventDemo.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaaproject.kaa.demo.event;
+
+import java.io.IOException;
+import java.lang.String;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.kaaproject.kaa.client.DesktopKaaPlatformContext;
+import org.kaaproject.kaa.client.Kaa;
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
+import org.kaaproject.kaa.client.event.EventFamilyFactory;
+import org.kaaproject.kaa.client.event.EventListenersResolver;
+import org.kaaproject.kaa.client.event.FetchEventListeners;
+import org.kaaproject.kaa.client.event.registration.EndpointRegistrationManager;
+import org.kaaproject.kaa.client.event.registration.UserAttachCallback;
+import org.kaaproject.kaa.client.transact.TransactionId;
+import org.kaaproject.kaa.common.endpoint.gen.UserAttachResponse;
+import org.kaaproject.kaa.schema.sample.event.thermo.ChangeDegreeRequest;
+import org.kaaproject.kaa.schema.sample.event.thermo.CustomThermoEventClassFamily;
+import org.kaaproject.kaa.schema.sample.event.thermo.CustomThermoEventClassFamily.DefaultEventFamilyListener;
+import org.kaaproject.kaa.schema.sample.event.thermo.ThermostatInfoRequest;
+import org.kaaproject.kaa.schema.sample.event.thermo.ThermostatInfoResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EventDemo {
+
+
+    private static final Logger LOG = LoggerFactory.getLogger(EventDemo.class);
+    public static void main(String[] args) {
+        LOG.info("Event demo has been started");
+        doWork();
+        LOG.info("Event demo has been stopped");
+    }
+
+
+    public static void doWork() {
+        KaaClient kaaClient = Kaa.newClient(new DesktopKaaPlatformContext(),
+                new SimpleKaaClientStateListener());
+        kaaClient.start();
+
+        EndpointRegistrationManager registrationManager = kaaClient
+                .getEndpointRegistrationManager();
+
+        registrationManager.attachUser("userExternalId", "userAccessToken",
+                new UserAttachCallback() {
+                    @Override
+                    public void onAttachResult(UserAttachResponse response) {
+                        LOG.info("Attach response"
```

Comment:
LOG.info("Attach response: {}", response.getResult());
Fix similar issues all over the code please